models/prostate_mri_anatomy/configs/inference.json

```diff
@@ -0,0 +1,153 @@
+        "in_channels": 1,
+        "out_channels": 3,
+        "channels": [
+            16,
+            32,
+            64,
+            128,
+            256,
+            512
+        ],
+        "strides": [
+            2,
+            2,
+            2,
+            2,
+            2
+        ],
+        "num_res_units": 4,
+        "norm": "batch",
+        "act": "prelu",
+        "dropout": 0.15
+    },
+    "network": "$@network_def.to(@device)",
+    "preprocessing": {
+        "_target_": "Compose",
+        "transforms": [
+            {
+                "_target_": "LoadImaged",
+                "keys": "image"
+            },
+            {
+                "_target_": "EnsureChannelFirstd",
+                "keys": "image"
+            },
+            {
+                "_target_": "Orientationd",
+                "keys": "image",
+                "axcodes": "RAS"
+            },
+            {
+                "_target_": "Spacingd",
+                "keys": "image",
+                "pixdim": [
+                    0.5,
+                    0.5,
+                    0.5
+                ],
+                "mode": "bilinear"
+            },
+            {
+                "_target_": "ScaleIntensityd",
+                "keys": "image",
+                "minv": 0,
+                "maxv": 1
+            },
+            {
+                "_target_": "NormalizeIntensityd",
+                "keys": "image"
+            },
+            {
+                "_target_": "EnsureTyped",
+                "keys": "image"
+            }
+        ]
+    },
+    "dataset": {
+        "_target_": "Dataset",
+        "data": "$[{'image': i} for i in @datalist]",
+        "transform": "@preprocessing"
+    },
+    "dataloader": {
+        "_target_": "DataLoader",
+        "dataset": "@dataset",
+        "batch_size": 1,
+        "shuffle": false,
+        "num_workers": 4
+    },
+    "inferer": {
+        "_target_": "SlidingWindowInferer",
+        "roi_size": [
+            96,
+            96,
+            96
+        ],
+        "sw_batch_size": 4,
+        "overlap": 0.5
+    },
+    "postprocessing": {
+        "_target_": "Compose",
+        "transforms": [
+            {
+                "_target_": "AsDiscreted",
+                "keys": "pred",
+                "argmax": true
+            },
+            {
+                "_target_": "KeepLargestConnectedComponentd",
+                "keys": "pred",
+                "applied_labels": [
+                    1,
+                    2
+                ]
+            },
+            {
+                "_target_": "SaveImaged",
+                "keys": "pred",
+                "resample": false,
+                "meta_keys": "pred_meta_dict",
+                "output_dir": "@output_dir"
+            }
+        ]
+    },
+    "handlers": [
+        {
+            "_target_": "CheckpointLoader",
+            "load_path": "$@bundle_root + '/models/anatomy.pt'",
```

Comment:
I think the saved model name in your `train.json` is `model.pt`.
Please update this line to be consistent.

Thanks.

Reply:
The model names are now consistent in all files. It's always `model.pt`